src/web-components/abstract/KSLCustomElementWithTooltip.ts

```diff
@@ -1,44 +1,34 @@
 import { KSLCustomElement } from './KSLCustomElement';
-import { ElementPosition } from './KSLPositionedElement';
 import { KSLTooltipElement } from '../KSLTooltipElement';
+import { ElementPositionOffset } from './KSLPositionedElement';
 
 export abstract class KSLCustomElementWithTooltip extends KSLCustomElement {
+  public static get observedAttributes(): string[] {
+    return ['tooltip-disabled', 'tooltip-message'];
```

Comment:
just curious, have you thought about constants?

Reply:
We discussed this on the call. We will leave attribute names as strings for now and possibly replace them with constants in the future.